tests/test_printing.py

```diff
@@ -41,18 +41,19 @@ def test_notex_print():
     )
     assert pprint(normalrv_noname_expr) == expected.strip()
 
+    # TODO: This is no longer working with Aesara==2.3.0
```

Comment:
The behavior changed because of https://github.com/aesara-devs/aesara/commit/3d4ef668cafce97acfab095e0c1a44752abefb2c

@brandonwillard Is there an easy fix?

Reply:
https://github.com/aesara-devs/aesara/pull/680 should fix this specific issue.